dora/core/client/fs/src/main/java/alluxio/client/file/cache/PageId.java

```diff
@@ -64,6 +69,16 @@ public boolean equals(Object obj) {
     return mFileId.equals(that.mFileId) && mPageIndex == that.mPageIndex;
   }
 
+  public static List<PageId> getAllPageIdsForFile(String path, long length, long pageSize) {
```

Comment:
I think we have added a very similar method in LocalCacheManager

Reply:
LocalCacheManager::getCachedPageIdsByFileId

Reply:
yeah switched to use that one